packages/rancher-monitoring/generated-changes/patch/templates/exporters/node-exporter/servicemonitor.yaml.patch

```diff
@@ -0,0 +1,8 @@
+--- charts-original/templates/exporters/node-exporter/servicemonitor.yaml
++++ charts/templates/exporters/node-exporter/servicemonitor.yaml
+@@ -1,4 +1,4 @@
+-{{- if .Values.nodeExporter.enabled }}
++{{- if (and .Values.nodeExporter.enabled .Values.nodeExporter.serviceMonitor.enabled) }}
```

Comment:
This should be disabled if PushProx is enabled since PushProx already comes in with a ServiceMonitor